ios/Samples/AndorraTelecomPublic/AndorraTelecomPublic/Resources/js/menu.js

```diff
@@ -40,6 +43,12 @@ function createButtonMap(text) {
     return containerButton;
 }
 
+function removeClassName(items, cssClass) {
+    for (var i = items.length - 1; i >= 0; i--) {
```

Comment:
:O any reason to do it backwards? it's slightly better in performance and used to be recommended but browsers are dealing with the 2 ways the same nowadays.

It's Ok, I'm just surprised :)